kotlin/authenticate-with-oauth/README.metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "category": "Cloud and Portal",
+  "description": "Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires logging in to the portal that hosts them (an ArcGIS Online account, for example).",
+  "ignore": false,
+  "images": [
+    "authenticate-with-oauth.png"
+  ],
+  "keywords": [
```

Comment:
I guess since we're here might as well update the keywords based on tag changes above?